packages/v4/src/content/developer-resources/accessibility-testing.md

```diff
@@ -0,0 +1,106 @@
+
+
+
+6. **Check color contrast for the following:**
+    - Text color against background color ([Understanding WCAG 1.4.3](//www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html))
+    - Text color against link color ([Technique G183](//www.w3.org/TR/WCAG20-TECHS/G183.html))
+    - Visible boundaries of buttons and form elements against adjacent background color ([Understanding WCAG 1.4.11](//www.w3.org/WAI/WCAG21/Understanding/non-text-contrast.html))
+
+
+
+## Screen Readers
+To truly test your accessibility, open up a screen reader of your choice (we use Voice Over). Just as front-end developers use their browser to see how their changes look, you should also open a screen reader to see how your accessibility looks. 
+
+Generally, screen readers access the DOM (Document Object Model), and they use browser APIs (Application Programming Interfaces) to get the information they need. In this way, a screen reader knows what to say when a set of list items begins and ends, and it typically announces, in advance, how many items are in the list. A screen reader can also traverse a page using heading navigation to speak the heading level. 
+
+Here are a few aspects that can affect how screen readers communicate information:
+
+- **Semantic HTML** - Semantics refers to the meaning of a piece of code. A semantic element clearly describes its meaning to both the browser and the developer. For example, `<div>` and `<span>` are non-semantic elements because it tells nothing about its contents. Examples of semantic elements include `<form>`, `<table>`, etc. which clearly define its contents. Screen readers expect semantic HTML when traversing the DOM so non-semantic elements that are not customized to be made accessible are highly likely to be inaccessible. Aria and other accessible attributes are meant to extend the functionality and meaning of native semantics, but at the core, your HTML should be semantic.
+
+- **Headings** - A visual user can scan a page and fairly quickly understand the information architecture of the page. Visually impaired users have other methods of achieving this. One common method is the use of headings where a user will use the heading levels to determine the flow of information. Having headings that vary in size simply based on design and not functionality will likely confuse these users. Having a clear flow of sequential heading sizes based on headings and subheadings is significantly clearer to all users.
+
+- **Accessible names for all elements** - When an element does not have visual text or when further explanation is necessary, a screen reader will not know what an item is or does. If you have an icon `<button>`, for example, the screen reader can only tell that it’s a button if it doesn’t have a label, not what the button does. Add a few more non-labeled buttons, and now all the screen reader sees are vague buttons with no understanding of what they do. See above section on labeling for more information.
```

Comment:
```suggestion
- **Accessible names for all elements**: When an element doesn't have visual text or when further explanation is necessary, a screen reader will not know what an item is or does. For example, if you have an icon `<button>`, and it doesn't have a label, the screen reader can only tell that it’s a button—it can't determine what the button does. Add a few more non-labeled buttons, and now all the screen reader sees are vague buttons with no understanding of what they do. For more information, see the labeling section.
``` 
Can we anchor link "labeling section" ?